spec/unit/check_board_spec.rb

```diff
@@ -1,38 +1,232 @@
 # frozen_string_literal: true
 
 describe CheckBoard do
-  it 'can return empty string for an empty board' do
-    check_board = CheckBoard.new
-    expect(check_board.execute).to eq('')
-  end
+  let(:check_board) { CheckBoard.new }
```

Comment:
That's okay but for the sake of learning cool tricks, check what `subject` in RSpec is 👍